src/Bottle/provider.js

```diff
@@ -93,6 +94,25 @@ var createProvider = function createProvider(name, Provider) {
     return this;
 };
 
+var removeProviderMap = function resetProvider(name) {
+    this.providerMap[name] = false;
```

Comment:
@cjsaylor I think you can delete this one too.  It's only set to `true` in the provider function so that the key exists with a truthy value.